common/djangoapps/course_modes/views.py

```diff
@@ -103,19 +103,23 @@ def get(self, request, course_id, error=None):
                     redirect_url = ecommerce_service.get_checkout_page_url(professional_mode.bulk_sku)
             return redirect(redirect_url)
 
+        course = modulestore().get_course(course_key)
```

Comment:
Any idea the extra load requirements of loading the course here?  I don't know modulestore very well.

Reply:
i removed it below and am loading it earlier in this function, but not more times, so i don't think there should be any extra load